docs/rfcs/XeTile.md

```diff
@@ -105,29 +99,16 @@ Attribute `padding` specifies the padding value for the out-of-boundary access.
      vector<64x32xb16> to tile<64x32xbf16, #tile_attr>
 ```
 
-`load_tile` stores a 4D vector to a 2D tile with an `inner_block`.
-```mlir
-  #tile_attr = #xetile.tile_ttr<inner_blocks=[16,16]>
-  %vector_a = XeTile.store_tile %tile_a :
-     vector<4x2x16x16xb16> into tile<64x32xbf16, #tile_attr>
-```
-
 `prefetch_tile` prefetches the tile to cache.  
 ```mlir
   XeTile.prefetch_tile %coop_tile: tile<8x32xbf16>
 ```
 `tile_mma` represents the matrix multiplication on 2D vectors. The semantics can be represented by vector.contract, so tile_mma works more like a syntax sugar. This also means that the code can be lowered to vector.contract and mapped to HW without DPAS support nicely.  
 ```mlir
-  %vector_c = XeTile.tile_mma %vector_a, %vector_b, %vector_c :
+  %vector_c = XeTile.tile_mma %vector_c, %vector_b, %vector_a :
```

Comment:
We were following the format of `tile_mma %a, %b, %c` for implementation, maybe it is better to keep it consistent.

Reply:
fixed